features/admin.users.v1/components/user-profile.tsx

```diff
@@ -1327,6 +1499,743 @@ export const UserProfile: FunctionComponent<UserProfilePropsInterface> = (
+                    message: t("user:profile.notifications.updateProfileInfo." +
+                        "genericError.message")
+                }));
+            })
+            .finally(() => {
+                setIsSubmitting(false);
+            });
+    };
+
+    /**
+     * Verify an email address or mobile number.
+     *
+     * @param schema - Schema of the attribute
+     * @param attributeValue - Value of the attribute
+     */
+    const handleVerify = (schema: ProfileSchemaInterface, attributeValue: string) => {
+        setIsSubmitting(true);
+        const data: PatchOperationRequest<PatchUserOperationValue> = {
+            Operations: [
+                {
+                    op: "replace",
+                    value: {}
+                }
+            ],
+            schemas: [ "urn:ietf:params:scim:api:messages:2.0:PatchOp" ]
+        };
+        let translationKey: string = "";
+
+        if (schema.name === EMAIL_ADDRESSES_ATTRIBUTE) {
+            translationKey = "user:profile.notifications.verifyEmail.";
+            const verifiedEmailList: string[] = profileInfo?.get(ProfileConstants.SCIM2_SCHEMA_DICTIONARY.
+                get("VERIFIED_EMAIL_ADDRESSES"))?.split(",") || [];
+
+            verifiedEmailList.push(attributeValue);
+            data.Operations[0].value = {
+                [schema.schemaId]: {
+                    [VERIFIED_EMAIL_ADDRESSES_ATTRIBUTE]:
+                        verifiedEmailList
+                }
+            };
+        } else if (schema.name === MOBILE_NUMBERS_ATTRIBUTE) {
+            translationKey = "user:profile.notifications.verifyMobile.";
+            setSelectedAttributeInfo({ schema, value: attributeValue });
+            const verifiedMobileList: string[] = profileInfo?.get(ProfileConstants.SCIM2_SCHEMA_DICTIONARY.
+                get("VERIFIED_MOBILE_NUMBERS"))?.split(",") || [];
+
+            verifiedMobileList.push(attributeValue);
+            data.Operations[0].value = {
+                [schema.schemaId]: {
+                    [VERIFIED_MOBILE_NUMBERS_ATTRIBUTE]:
+                        verifiedMobileList
+                }
+            };
+        }
+
+        setIsSubmitting(true);
+        updateUserInfo(user.id, data)
+            .then(() => {
+                onAlertFired({
+                    description: t(
+                        `${translationKey}success.description`
+                    ),
+                    level: AlertLevels.SUCCESS,
+                    message: t(
+                        `${translationKey}success.message`
+                    )
+                });
+
+                handleUserUpdate(user.id);
+            })
+            .catch((error: AxiosError) => {
+                if (error?.response?.data?.detail || error?.response?.data?.description) {
+                    dispatch(addAlert({
+                        description: error?.response?.data?.detail || error?.response?.data?.description,
```

Comment:
We have shown the description in other places as well, I think this is useful in some cases, since console is used by Admins it's okay to show this kind of errors. For example, an LDAP-specific error message provides more insight than a generic error, guiding admins to address attribute compatibility.

https://github.com/PasinduYeshan/wso2-identity-apps/blob/22bdbde1b16ec65279643b7b03d5c7e6b5986c30/features/admin.users.v1/components/user-profile.tsx#L1189-L1194